content/en/integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose.md

```diff
@@ -29,17 +29,18 @@ Using Amazon CloudWatch Metric Streams and Amazon Kinesis Data Firehose, you can
 2. Once you create these resources, Datadog immediately starts receiving the streamed metrics and displays them in the Datadog application with no additional configuration needed.
-   - **Note**: Per-namespace defaults and account-level settings in the AWS Integration tile only apply to the API polling approach. Manage all rules for including and excluding namespaces in the streams using the CloudWatch Metric Streams configuration in your AWS accounts.
-   - If you already receive metrics for a given CloudWatch namespace through the API polling method, Datadog automatically detects this and stops polling metrics for that namespace since you are now streaming them. This could lead to some differences in the specific metric names collected as noted in [Supported Metrics](?tab=cloudformation#supported-metrics).
-   - If you later decide you don't want to stream a metric and delete the stream or remove namespaces from it, Datadog automatically starts collecting those metrics using API polling again, according to your configuration settings in the AWS Integration tile.
- 
-### Supported metrics
-Nearly all of the CloudWatch namespaces and metrics that Datadog supports via the API polling approach are also supported via Metric Streams. There are however a couple exceptions.
+   
+   
+### API Polling vs Metric Streaming 
 
-The following CloudWatch metrics are not currently supported:
+   - **Namespace filtering on AWS**: Per-namespace defaults and account-level settings in the AWS Integration tile only apply to the API polling approach. Manage all rules for including and excluding namespaces in the streams using the CloudWatch Metric Streams configuration in your AWS accounts.
+   - **Metrics Collected**: If you already receive metrics for a given CloudWatch namespace through the API polling method, Datadog automatically detects this and stops polling metrics for that namespace since you are now streaming them. This could lead to some differences in the specific metric names collected. For example: __ 
+   - **S3 daily storage metrics and Billing metrics** like ```aws.s3.bucket_size_bytes``` and ```aws.billing.estimated_charges``` will continue being collected through the API polling method even after metric streaming is enabled.
+   - **Fallback to API Polling**: If you later decide you don't want to stream a metric and delete the stream or remove namespaces from it, Datadog automatically starts collecting those metrics using API polling again, according to your configuration settings in the AWS Integration tile.
+ 
+ <div class="alert alert-warning">Nearly all of the CloudWatch namespaces and metrics that Datadog supports via the API polling method are also supported via Metric Streams. The only exception is CloudWatch metrics for <a href="https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/cloudwatch_concepts.html#Percentiles" target="_blank">percentile statistics</a> (p90, p95, p99, etc.) which is currently not supported by streaming.
```

Comment:
A few things on this one:

1. Let's move this to be a new bullet at the very top of the "API Polling vs Metric Streaming" section, and keep the alert format. I want this to be the first thing customers see when comparing.
2. Let's reframe this bullet and clearly call it "Unsupported Metrics" or something like and remove the positive "Nearly all of the CloudWatch namespaces and metrics that Datadog supports..." beginning. Let's just directly go to what is not supported.
3. Move the examples of what is not supported to this section